src/messages/discord_embeds.py

```diff
@@ -16,7 +16,8 @@ def __init__(self):
         try:
             self._webhook = Webhook.from_url(Config().general['custom_message_hook'], adapter=RequestsWebhookAdapter(), )
         except InvalidArgument:
-            Logger.warning(f"Your custom_message_hook URL {Config().general['custom_message_hook']} is invalid, Discord updates will not be sent")
+            if Config().general["custom_message_hook"]:
```

Comment:
```suggestion
              Logger.error("Message about error")
```

Reply:
@VladimirMakaev the trouble is it sends the error multiple times for some reason. I could put a debug line saying "not set" but then it'd spam it multiple times. I've rearranged the logic slightly in latest commit but otherwise I think it should be fine.